### PR TITLE
fix(roots option): Add roots option for jest v19

### PR DESCRIPTION
**Pull request by nicojs**

### CODE REVIEW

**Comment by nicojs** [comment]
For some reason jest v19 didn't work for me because the `roots` option wasn't set.

File: .vscode/launch.json [hunk 1]
```diff
@@ -1,123 +1,24 @@
 {
```
Comment (mthmulders) [inline .vscode/launch.json]: Why did you remove most of the configurations?

File: test/integration/JestTestRunner.it.ts [hunk 2]
```diff
@@ -7,165 +7,188 @@ chai.use(chaiAsPromised);
...
-        before(() => {
-            testRunnerOptions = {
-                files: [
-                    file({ name: 'testResources/sampleProject/src/Add.js', mutated: true }),
-                    file({ name: 'testResources/sampleProject/src/__tests__/AddSpec.js', mutated: false })],
-                port: 9877,
-                strykerOptions: { logLevel: 'trace' }
-            };
-        });
+  let sut: JestTestRunner;
```
Comment (mthmulders) [inline test/integration/JestTestRunner.it.ts]: Do we really need to re-format the whole file because of an `r`? If it must be re-formatted, could you _please_ do that in a separate commit?

File: testResources/roots-option-project/__tests__/isAdult.spec.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const isAdult = require('../lib/isAdult.js');
```
Comment (mthmulders) [inline testResources/roots-option-project/__tests__/isAdult.spec.js]: Cool to see an extra test that verifies the fix! Will it work with Jest 18?

File: .vscode/launch.json (hunk 1 above)
Comment (nicojs) [reply .vscode/launch.json]: Didn't know you used it ;). It gave deprecation warnings, so i decided to regenerate it. I don't mind either way, shall i undo this change?

File: test/integration/JestTestRunner.it.ts (hunk 2 above)
Comment (nicojs) [reply test/integration/JestTestRunner.it.ts]: The `r` change was a different file. I'm adding an integration test here. Also: an indent 2 is in `.editorconfig`, so i think thats why my editor forced this reformatting. Shall i undo the formatting for this file?

File: test/integration/JestTestRunner.it.ts [hunk 4]
```diff
@@ -7,165 +7,188 @@ chai.use(chaiAsPromised);
...
+          file({ name: 'testResources/sampleProject/src/__tests__/AddSpec.js', mutated: false }),
+          file({ name: 'testResources/sampleProject/src/Error.js', mutated: false, included: false })],
+        port: 9881,
+        strykerOptions: {}
+      };
+      sut = new JestTestRunner(testRunnerOptions);
+      return sut.init();
+    });
 
-        it('should report Complete without errors', () => {
```
Comment (nicojs) [inline test/integration/JestTestRunner.it.ts]: I see what you mean... this isn't fun. I will undo my change.

File: testResources/roots-option-project/__tests__/isAdult.spec.js (hunk 3 above)
Comment (nicojs) [reply testResources/roots-option-project/__tests__/isAdult.spec.js]: I believe so: https://travis-ci.org/stryker-mutator/stryker-jest-runner/jobs/296137148

File: .vscode/launch.json (hunk 1 above)
Comment (mthmulders) [reply .vscode/launch.json]: Please do. I'll see if I get the deprecation warnings, too, and then update it. My idea would be to have some ready-to-go launch configs, so anyone who would be working on this project could immediately get started.

Comment (nicojs) [reply .vscode/launch.json]: Done

**Comment by nicojs** [comment]
Everything should be fine now.

**Comment by mthmulders** [comment]
Thanks, looks better now. We can easily see what has changed, and what hasn't. Looks good to me!
